rdrf/rdrf/forms/navigation/quick_links.py

```diff
@@ -402,27 +360,28 @@ def _build_menu(self):
                 **Links.OTHER,
             }
             # menu with everything, used for the admin page
-            if settings.DESIGN_MODE:
+            if is_design_mode():
                 design_menus.update(proms_menus)
-                MenuConfig().all = design_menus
+                self.menu_config.all = design_menus
             else:
-                MenuConfig().all = proms_menus
+                self.menu_config.all = proms_menus
 
     def _group_links(self, group):
```

Comment:
Is this used anywhere? Even if it is, why isn't this a method on `MenuConfig` instead?

Reply:
Yes. It's used in the menu_links method. Moved it to MenuConfig.